src/org/centos/pipeline/PipelineUtils.groovy

```diff
@@ -1333,15 +1333,15 @@ def injectArray(String prefix, def message) {
 def repoFromRequest(String request, String prefix) {
 
     try {
-        def gitMatcher = request =~ /git.+?\/([a-z0-9A-Z_\-]+)(\.git|\?).*/
-        def cliMatcher = request =~ /cli-build.+?\/([a-zA-Z0-9\-_]+)-.*/
+        def gitMatcher = request =~ /git.+?\/([a-z0-9A-Z_\-]+)(?:\.git|\?).*/
+        def buildMatcher = request =~ /(?:koji-shadow|cli-build).+?\/([a-zA-Z0-9\-_]+)-.*/
```

Comment:
No idea if koji-shadow is just for the non primary koji instances, but good to have the code to support it if not. +1